pkg/varlinkapi/images.go

```diff
@@ -48,10 +57,188 @@ func (i *LibpodAPI) ListImages(call ioprojectatomicpodman.VarlinkCall) error {
 	return call.ReplyListImages(imageList)
 }
 
+// GetImage returns a single image in the form of a ImageInList
+func (i *LibpodAPI) GetImage(call ioprojectatomicpodman.VarlinkCall, name string) error {
+	runtime, err := libpodruntime.GetRuntime(i.Cli)
+	if err != nil {
+		return call.ReplyRuntimeError(err.Error())
+	}
+	newImage, err := runtime.ImageRuntime().NewFromLocal(name)
+	if err != nil {
+		return call.ReplyErrorOccurred(err.Error())
+	}
+	labels, _ := newImage.Labels(getContext())
+	containers, _ := newImage.Containers()
+	size, _ := newImage.Size(getContext())
```

Comment:
Safe to ignore errors on these 3? We deference the pointers later, this could be a segfault

Reply:
i think it is safe but ill catch them just in case.